From 9bad7f0d876a0ed4f7ee9fdff6ca6d117debf404 Mon Sep 17 00:00:00 2001
From: Pavel Vomacka <pvomacka@redhat.com>
Date: Thu, 11 Aug 2016 18:53:55 +0200
Subject: [PATCH] Add 'trusted to auth as user' checkbox

Add new checkbox to host and service details page

Prerequisite for: https://fedorahosted.org/freeipa/ticket/5764

Reviewed-By: Stanislav Laznicka <slaznick@redhat.com>
---
 install/ui/src/freeipa/host.js    | 5 +++++
 install/ui/src/freeipa/service.js | 5 +++++
 2 files changed, 10 insertions(+)

diff --git a/install/ui/src/freeipa/host.js b/install/ui/src/freeipa/host.js
index 33d443c2bc96c385bd13abf4d85adda6e51db718..87cf264ef20b79aceed639f45d926fd7aef19edf 100644
--- a/install/ui/src/freeipa/host.js
+++ b/install/ui/src/freeipa/host.js
@@ -142,6 +142,11 @@ return {
                             flags: ['w_if_no_aci']
                         },
                         {
+                            name: 'ipakrboktoauthasdelegate',
+                            $type: 'checkbox',
+                            acl_param: 'krbticketflags'
+                        },
+                        {
                             name: 'ipaassignedidview',
                             $type: 'link',
                             label: '@i18n:objects.idview.ipaassignedidview',
diff --git a/install/ui/src/freeipa/service.js b/install/ui/src/freeipa/service.js
index 35d486605ebfee41d8b3ffa5bb77bf9e72a60c01..30e336c35b8eece2e5e3ef55629d0c98f097fbf5 100644
--- a/install/ui/src/freeipa/service.js
+++ b/install/ui/src/freeipa/service.js
@@ -142,6 +142,11 @@ return {
                             acl_param: 'krbticketflags'
                         },
                         {
+                            name: 'ipakrboktoauthasdelegate',
+                            $type: 'checkbox',
+                            acl_param: 'krbticketflags'
+                        },
+                        {
                             name: 'ipakrbrequirespreauth',
                             $type: 'checkbox',
                             acl_param: 'krbticketflags'
-- 
2.7.4

